sections/semantics-sections.include

```diff
@@ -1128,6 +1128,48 @@
         </main>
       </body>
     </xmp>
+
+    The next example shows a misuse of a <{header}>, as its content does not <a>represent</a> an
+    introduction to its parent <a>sectioning content</a>:
+
+    <xmp highlight="html">
+      <article>
+        <h2>Spinning Straw into Gold</h2>
+        <p>A primer on getting rich quick, while avoiding making horrible promises.</p>
+
+        ...
+
+        <header>
+          <h3>Make a point to learn obscure names</h3>
+        </header>
+        ...
+      </article>
+    </xmp>
+
+    Modifying the previous example, the following markup better aligns to the appropriate usage
+    for a <{header}>:
+
+    <pre highlight="html">
+      &lt;article>
+        <mark>&lt;header></mark>
+          &lt;h2>Spinning Straw into Gold&lt;/h2>
+          &lt;p>A primer on getting rich quick, while avoiding making horrible promises.&lt;/p>
+        <mark>&lt;/header></mark>
+        ...
+
+        <mark>&lt;section></mark>
+          &lt;header>
+            &lt;h3>Make a point to learn obscure names&lt;/h3>
+          &lt;/header>
+          ...
+        <mark>&lt;/section></mark>
+      &lt;/article>
+    </pre>
+
+    ...now that content that appropriately introduced the <{article}> is contained within a
+    <{header}>, and the content that was previously in a <code>header</code> is now appropriately
+    scoped to a subsection of the <code>article</code>.
```

Comment:
I would put all this explanatory text with the previous bit. Your mileage may vary...